### ISSUE TITLE
Don't seem to be getting any errors, but can't access my server

### ISSUE THREAD

**Issue by jem-computer** [arunoda/meteor-up-legacy#252]
Sorry if I'm just being dumb - I can't figure out how to get this running though.

Brand new DigitalOcean image; was going to host Mongo locally rather than using Compose ($) -

```
{
  // Server authentication info
  "servers": [
    {
      "host": "100.100.100.100", // replaced with actual IP
      "username": "root",
      "pem": "~/.ssh/id_rsa"
    }
  ],


  "setupMongo": true,

  "setupNode": true,

  "nodeVersion": "0.10.33",

  "setupPhantom": true,

  "appName": "stories",

  "app": ".",

  "env": {
    "ROOT_URL": "http://100.100.100.100",
    "PORT": "3001"
  },

  "deployCheckWaitTime": 30
}
```

afaik from reading the docs, it should just figure out the local mongo URL by itself?

Deploying works with no errors - `mup logs -f`

```
Meteor Up: Production Quality Meteor Deployments
------------------------------------------------

 >> stepping down to gid: meteoruser
 >> stepping down to uid: meteoruser
```

```
$ ssh 100.100.100.100
$ curl localhost:3001

<!DOCTYPE html>
<html>
<head>
  <link rel="stylesheet" type="text/css" class="__meteor-css__" href="/48548b5b43343321ec7c6345b2bdbe36efc32205.css?meteor_css_resource=true">


<script type="text/javascript">__meteor_runtime_config__ = {"meteorRelease":"METEOR@1.0.2.1","ROOT_URL":"http://188.166.39.157","ROOT_URL_PATH_PREFIX":"","autoupdateVersion":"31654ccc53a11b3e4347a118bd7283bb65f88db7","autoupdateVersionRefreshable":"cd19ee7f9583e501a4263ce72bb0dd2ded800356","autoupdateVersionCordova":"d100f5651b90251c2e24ae6c4fa6013735ea8523"};</script>

  <script type="text/javascript" src="/233e6b641a4cb465740c3b40b04fff1102f3a711.js"></script>
<title>My Meteor App</title>
</head>
<body>

</body>
</html>
```

So it looks like it's all working & serving.

But then in my browser, going to `100.100.100.100:3001` just says 'this webpage is not available.

Not sure what to do next. Not sure if the nginx bit in the guide is optional or not - do I need to set it up even if I'm just running one meteor app on this server?

**Comment by davidrinnan** [comment]
No idea really. :) 
Are you maybe just having an issue with port 3001 not being available from the outside? 

what happens if you run it on port 80 instead, so long as you dont have anything else on that port that is. 

What happens if you do this in a js file and run it with node. Set the port to let's say 3002. can you reach if from the outside? 

// Load the http module to create an http server.
var http = require('http');

// Configure our HTTP server to respond with Hello World to all requests.
var server = http.createServer(function (request, response) {
  response.writeHead(200, {"Content-Type": "text/plain"});
  response.end("Hello World\n");
});

// Listen on port 8000, IP defaults to 127.0.0.1
server.listen(8000);

// Put a friendly message on the terminal
console.log("Server running at http://127.0.0.1:8000/");

**Comment by jem-computer** [comment]
Yeah, that works. I'm kind of stumped; I don't know enough about how Meteor runs as a Node process to figure out why it's not picking up.

I saw in #202 that force-ssl might be the problem; tried disabling it, no luck.

From Sacha's video it looked like you don't need nginx to be setup for things to run? What else would cause the port to not be open?

**Comment by jonwhittlestone** [comment]
I am having a similar issue. Have deployed a Meteor app with no problems using mup. Then on the same Linode machine - with port 3001 - no joy. I am using an nginx vhost.

[SO Post - Hosting multiple meteor apps on Linode with mup](http://stackoverflow.com/questions/29021863/hosting-multiple-meteor-apps-on-linode-with-mup)
